datafusion/proto/build.rs

```diff
@@ -61,8 +61,9 @@ fn build() -> Result<(), String> {
     let json = std::fs::read_to_string(out.join("datafusion.serde.rs")).unwrap();
     let mut file = std::fs::OpenOptions::new()
         .write(true)
+        .truncate(true)
```

Comment:
This was key to use just one output file for either feature set and avoid combinatorial explosions.